native/src/modules/categories/components/CategoryListContent.js

```diff
@@ -31,19 +32,14 @@ type ContentPropsType = {|
   theme: ThemeType
 |}
 
-const textDistanceToBullet = 10
-const listIndent = 20
-const bulletSizeRelativeToFont = 2.8
-const bulletAlignmentRelativeToFont = 2
-
 const CategoryListContent = ({
-  content,
-  navigateToLink,
-  cacheDictionary,
-  language,
-  lastUpdate,
-  theme
-}: ContentPropsType) => {
+                               content,
+                               navigateToLink,
+                               cacheDictionary,
+                               language,
+                               lastUpdate,
+                               theme
+                             }: ContentPropsType) => {
```

Comment:
Indentation

Reply:
This

> According to this it seems like a beta/stable release is not really far away as just? docs and code coverage is missing:
> [meliorence/react-native-render-html#434 (comment)](https://github.com/meliorence/react-native-render-html/pull/434#issue-527747509)

That's also what I expect. However this could mean 1 month or 3 month still. Therefore this PR proposal.